src/main/java/javaslang/collection/HashArrayMappedTrie.java

```diff
@@ -17,7 +17,7 @@
 /**
  * <a href="https://en.wikipedia.org/wiki/Hash_array_mapped_trie">Hash array mapped trie (HAMT)</a>.
  */
-abstract class HashArrayMappedTrie<K, V> implements Iterable<Tuple2<K, V>> {
+abstract class HashArrayMappedTrie<K, V> implements Iterable<Tuple2<K, V>>, Serializable {
 
```

Comment:
Yes, I see, fixed.
But there is another error message when I do <code>mvn test</code>:
```text
Failed tests:   shouldHaveAConsistentTypeSystem(javaslang.TypeConsistencyTest): Unoverriden methods found. Check if this is ok and copy & paste the necessary lines into the TypeConsistencyTest.WHITELIST:
```